lib/matplotlib/axes/_axes.py

```diff
@@ -2770,6 +2772,14 @@ def errorbar(self, x, y, yerr=None, xerr=None,
             every 5-th datapoint will be plotted. The data plot itself still
             shows all data points.
 
+        border : string, optional, default: None
+            A string denoting the desired color of the border of the errorbar.
+            Default, no border is drawn around the errorbar.
+
+        borderwidth : positive integer, optional, default:3
```

Comment:
To my knowledge I do not think there is a reasonable way to combine border and borderwidth. It is because Stroke (from patheffects) is used where the arguments 'linewidth' and 'foreground' are separated.